src/components/VideosList.vue

```diff
@@ -11,7 +11,7 @@
                     <p class="text-sm font-medium text-near-green text-center">
                       {{ video.collection }}
                     </p>
-                    <p class="text-2xl font-semibold text-white">
+                    <p class="text-2xl font-semibold text-white w-64 text-center">
```

Comment:
If we use this style, we should be careful about very long video names because long video names will still overflow. 
eg. `near validators current near validators near validators current near validators`. You can try it on the dom.
However, I don't think there will be that long video names tho :D

Reply:
you can set font-size based on text length; just check the length, and make decisions on where those font-size breakpoints should be, e.g.` length > 50 ? font-size: 1.2rem : ...`

Reply:
yeah, also I think Airtable itself will limit the text length in the title.